auth4genai/snippets/get-started/langchain-next-js/async-auth.mdx

```diff
@@ -95,43 +95,142 @@ Next, add the following code to `src/lib/auth0.ts`:
 
 ```tsx src/lib/auth0.ts wrap lines highlight={2-5}
```

Comment:
highlight lines would need update, can you plz update that for all updated code blocks